core/src/main/scala/com/pingcap/tispark/write/TiDBOptions.scala

```diff
@@ -282,6 +286,7 @@ object TiDBOptions {
   val TIDB_ENABLE_UPDATE_TABLE_STATISTICS: String = newOption("enableUpdateTableStatistics")
   val TIDB_DEDUPLICATE: String = newOption("deduplicate")
   val TIDB_ROWID: String = newOption("tidbRowId")
+  val TiDB_ROW_FORMAT_VERSION: String = newOption("rowFormatVersion")
```

Comment:
how can insert sql pass this config?

Reply:
users need to set it by `spark.conf.set()`. I'll add it  in `newWriteBuilder `

Reply:
yes. you need to convert any configs users may set to tidboption in newWriteBuilder. And these configs should be in your doc